src/payment/strategies/adyenv2/adyenv2-payment-strategy.ts

```diff
@@ -315,4 +325,15 @@ export default class AdyenV2PaymentStrategy implements PaymentStrategy {
     private _updateAdyenComponentState(newState: AdyenCardState) {
         this._adyenComponentState = newState;
     }
+
+    private _getBrowserInfo(): BrowserInfoRequest {
+        return {
+            language: navigator.language || navigator.languages[0],
```

Comment:
As mentioned by @hx, you'll need call `navigator.userLanguage` for IE11 users. I don't think `navigator.languages` is supported by IE as well. You will get a compiler error if you write `userLanguage` because it's a non-standard property (TS only has types for standard properties). So you will need to write something like `navigator.language || (navigation as any).userLanguage` or define your own type.